### ISSUE TITLE
Dark mode

### ISSUE THREAD

**Issue by mcrascal** [evidence-dev/evidence#750]
Global dark mode: 

- Toggle in header (defaults to system, options for dark or light) 
- Dark presets in echarts theme

**Comment by nidhi-kala** [comment]
If this is available, I can begin working on it. Might need a few pointers on the overall design (colors, toggle button design etc) for dark mode.

**Comment by mcrascal** [comment]
@nidhi-kala, this one's not quite ready to go yet -- I think we should tackle this after #738

**Comment by nidhi-kala** [comment]
Sounds good. 👍

**Comment by zachstence** [comment]
Closing in favor of https://github.com/evidence-dev/evidence/issues/2499
